packages/integrations/vercel/src/edge/adapter.ts

```diff
@@ -151,6 +151,11 @@ export default function vercelEdge({
 					version: 3,
 					routes: [
 						...getRedirects(routes, _config),
+						{
+							src: '^/_astro/(.*)$',
```

Comment:
The `_astro` path can be configured through [`build.assets`](https://docs.astro.build/en/reference/configuration-reference/#buildassets), it would be nice to derive this string from that too.